addons/account/models/account_move.py

```diff
@@ -1902,7 +1902,7 @@ def _validate_taxes_country(self):
     def _is_eligible_for_early_payment_discount(self, currency, reference_date):
         self.ensure_one()
         if not reference_date:
-            return True
+            return None
```

Comment:
```suggestion
            return
```
return is None by default

Reply:
Hello @JulienAlardot 
Changes are done.
Please check Thank You!!